index.html

```diff
@@ -656,12 +656,23 @@ <h4>Amanda Jepson</h4>
 
         </div>
           <div class="row"> 
-            <!--new change-->
-            <!-- awesome code about me-->
-            <!--charlesmotaroki@students.uonbi.ac.ke	-->
-            <!--wanjaerica@gmail.com	-->
-            <!--mboyaivy@gmail.com-->
-            <!--machoguabednego6@gmail.com	-->

```

Comment:
You were supposed to include your code beneath the commented line of your name and not remove anyone else's name...